google/cloud/bigquery/v2/minimal/internal/json_utils.cc

```diff
@@ -93,6 +93,19 @@ void ToJson(std::chrono::system_clock::time_point const& field,
   j[name] = std::to_string(m);
 }
 
+void removeEmptyArraysAndObjects(nlohmann::json& j) {
```

Comment:
I would have defined this function as:

```cc
nlohmann::json RemoveEmpty(nlohmann::json j) {
```

That way you can say:

```cc
return RemoveEmptyArraysAndObjects(nlohmann::json::parse( ... ));
```

Reply:
Done.